lib/MusicBrainz/Server/Controller/Label.pm

```diff
@@ -117,11 +119,29 @@ sub show : PathPart('') Chained('load')
     $c->model('MediumFormat')->load(map { $_->all_mediums } @$releases);
     $c->model('ReleaseLabel')->load(@$releases);
  
+    my @renamed_from = map { $_->target }
+                        grep { $_->direction == $MusicBrainz::Server::Entity::Relationship::DIRECTION_BACKWARD }
+                        grep { $_->link->type->gid eq 'e6159066-6013-4d09-a2f8-bc473f21e89e' } @{ $label->relationships };
```

Comment:
suggesting to store the first grep in a var, then use it for both `@renamed_from` and `@renamed_to`.
```suggestion
    my @rename_rels = grep { $_->link->type->gid eq 'e6159066-6013-4d09-a2f8-bc473f21e89e' } @{ $label->relationships };
    my @renamed_from = map { $_->target }
                        grep { $_->direction == $MusicBrainz::Server::Entity::Relationship::DIRECTION_BACKWARD }
                        @rename_rels;
```